### PR TITLE
Updates versions and fixes snackbar test

### PR DESCRIPTION
**Pull request by JoseAlcerreca**

### CODE REVIEW

File: todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java [hunk 1]
```diff
@@ -22,7 +22,9 @@
 import android.support.test.runner.AndroidJUnit4;
 import android.test.suitebuilder.annotation.LargeTest;
 
+import com.example.android.architecture.blueprints.todoapp.Injection;
```
Comment (florina-muntenescu) [inline todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java]: are these two new imports needed?

File: todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java [hunk 2]
```diff
@@ -73,16 +75,15 @@ public void registerIdlingResource() {
 
     @Test
     public void errorShownOnEmptyTask() {
-        // Add task title and description
+        // Add invalid title and description combination
```
Comment (florina-muntenescu) [inline todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java]: you don't need to do any type or close the keyboard to show an empty task. the click on the fab button can be performed directly.

File: todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java [hunk 3]
```diff
@@ -73,16 +75,15 @@ public void registerIdlingResource() {
...
                 closeSoftKeyboard());
-        // Save the task
+        // Try to save the task
         onView(withId(R.id.fab_edit_task_done)).perform(click());
 
-        // Verify empty tasks snackbar is shown
-        String emptyTaskMessageText = getTargetContext().getString(R.string.empty_task_message);
-        onView(withText(emptyTaskMessageText)).check(matches(isDisplayed()));
+        // Verify that the activity is still displayed (a correct task would close it).
+        onView(withId(R.id.add_task_title)).check(matches(isDisplayed()));
```
Comment (florina-muntenescu) [inline todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java]: Here you're verifying that the activity is still displayed. The test is called "errorShownOnEmptyTask" but there's no verification that an error is shown.

File: todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java [hunk 4]
```diff
@@ -73,16 +75,15 @@ public void registerIdlingResource() {
...
         onView(withId(R.id.add_task_title)).perform(typeText(""));
         onView(withId(R.id.add_task_description)).perform(typeText(""),
                 closeSoftKeyboard());
-        // Save the task
+        // Try to save the task
         onView(withId(R.id.fab_edit_task_done)).perform(click());
 
-        // Verify empty tasks snackbar is shown
-        String emptyTaskMessageText = getTargetContext().getString(R.string.empty_task_message);
-        onView(withText(emptyTaskMessageText)).check(matches(isDisplayed()));
```
Comment (florina-muntenescu) [inline todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java]: @JoseAlcerreca Why remove the check for the error display?

File: todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java (hunk 2 above)
Comment (JoseAlcerreca) [inline todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java]: Yeah but the test reads better.

File: todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java (hunk 1 above)
Comment (JoseAlcerreca) [inline todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java]: Nope, fixing

File: todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java (hunk 3 above)
Comment (JoseAlcerreca) [inline todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java]: Done

File: todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java (hunk 2 above)
Comment (JoseAlcerreca) [inline todoapp/app/src/androidTestMock/java/com/example/android/architecture/blueprints/todoapp/addedittask/AddEditTaskScreenTest.java]: Changed it for something even better

**Comment by JoseAlcerreca** [comment]
PTAL

**Pull request merged**